src-tauri/tauri.conf.json

```
@@ -0,0 +1,71 @@
+{
+  "$schema": "../node_modules/@tauri-apps/cli/schema.json",
+  "build": {
+    "beforeBuildCommand": "BUILD_TAURI=1 pnpm run build",
+    "beforeDevCommand": "BUILD_TAURI=1 pnpm run dev",
```

```suggestion
    "beforeBuildCommand": "pnpm cross-env BUILD_TAURI=1 pnpm run build",
    "beforeDevCommand": "pnpm cross-env BUILD_TAURI=1 pnpm run dev",
```
Consider using cross-env for Windows compatibility?

---

@mikelxk I use the `TAURI_PLATFORM` environment variable to resolve the provider module, it should work cross-platform. 

Also, I fixed some errors when running `pnpm tauri build`, it may still fail, if you have trouble with it, please let me know.

---

Thx for the previous fix!
However, I still encounter error when I ran `pnpm tauri build`
in `tauri.conf.json`, shouldn't the `distDir` be `../dist` instead of `./dist`？
I ran latter with no problem while `./dist` is not working

---

Yes, the distDir should be `../dist`

It still can't build on my laptop, and I think it should be an os independent issue, and I rush to work in the morning, and forgot to push the fix.